src/lightmap.cpp

```diff
@@ -571,44 +568,60 @@ void map::build_seen_cache()
             //
             // The naive solution of making the mirrors act like a second player
             // at an offset appears to give reasonable results though.
```

Comment:
random idea, but for this why not just cast the light like you usually do, but then if you hit a tile with a flag (i.e. IS_MIRROR), you simply continue with a reversed slope, so it will be from the player's point of view? or is this not reasonable?